DataConnectors/Forcepoint Cloud Security Gateway.json

```diff
@@ -0,0 +1,118 @@
+        },
+      {
+            "description" : "Top 5 Web Users with 'Action' equal to 'Blocked'",
+            "query": "CommonSecurityLog\n| where TimeGenerated  <= ago(0m)\n| where DeviceVendor == \"Forcepoint CSG\"\n| where DeviceProduct == \"Web\"\n| where Activity == \"Blocked\"\n| where SourceUserID != \"Not available\"\n| summarize Count=count() by SourceUserID\n| top 5 by Count\n| render piechart"
+        },
+        {
+            "description" : "Top 5 Sender Email Addresses Where Spam Score Greater Than 10.0",
+            "query": "CommonSecurityLog\n| where TimeGenerated  <= ago(0m)\n| where DeviceVendor == \"Forcepoint CSG\"\n| where DeviceProduct == \"Email\"\n| where DeviceCustomFloatingPoint1 > 10.0\n| summarize Count=count() by SourceUserName\n| top 5 by Count\n| render barchart"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (Forcepoint CSG)",
+            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CSG\"\n| where  DeviceProduct == \"Web\"\n  | summarize Time = max(TimeGenerated)\n  | where isnotempty(Time)"
+        },
+        {
+            "name": "CommonSecurityLog (Forcepoint CSG)",
+            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CSG\"\n| where  DeviceProduct == \"Email\"\n  | summarize Time = max(TimeGenerated)\n  | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CSG\"\n| where  DeviceProduct == \"Web\"\n  | summarize LastLogReceived = max(TimeGenerated)\n  | project IsConnected = LastLogReceived > ago(30d)",
+                "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint CSG\"\n| where  DeviceProduct == \"Email\"\n  | summarize LastLogReceived = max(TimeGenerated)\n  | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "Linux Syslog agent configuration",
+            "description": "This integration requires the Linux Syslog agent to collect your Forcepoint Cloud Security Gateway Web/Email logs as Common Event Format (CEF) and forward them to Azure Sentinel.",
```

Comment:
CEF connector should communicate via some port (514 TCP default).Please refer template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json